Structs/Structs/Order Struct/task.md

```diff
@@ -0,0 +1,13 @@
+## Address all the TODOs to make the tests pass!
+
+Complete the TODO field to make the tests pass!
+
```

Comment:
We do not specify the actual thing that is to be done, and do not reveal the tests, so it could be a bit confusing. We might state here what we really expect to get in this struct to pass the tests.